conf/scylla.yaml

```diff
@@ -245,7 +245,10 @@ batch_size_fail_threshold_in_kb: 50
 # broadcast_rpc_address: 1.2.3.4
 
 # Uncomment to enable experimental features
-# experimental: true
+# experimental:
+#     - cdc
+#     - lwt
+#     - udf
 
```

Comment:
Is there a way to keep supporting `experimental: true` to avoid breaking scripts?

Reply:
(it should just enable the entire set)

Reply:
In YAML that might prove difficult, because our YAML parser expects a sequence for `vector`-typed properties.  At best, we can support `experimental:\n  - true` (which would still break scripts).  Or we can have two properties: a binary `experimental` and a vector `experimental_foo`.

In CLI we have a bit more flexibility in implementation, so we could support things like `--experimental=all` or any of the backward-compatible boolean tokens (`on`, `yes`, `1`, and `true`).

Reply:
We could have two different options: "experimental: true" enables everything, while "experimental_feature: ..." enables just a list of specific features.
This will also allow us to keep the "--experimental" with no options.

Reply:
That's what I was saying above, except with "foo" instead of "feature".